django/db/backends/postgresql/introspection.py

```diff
@@ -22,15 +23,45 @@ class DatabaseIntrospection(BaseDatabaseIntrospection):
         700: 'FloatField',
         701: 'FloatField',
         869: 'GenericIPAddressField',
+        1000: ('django.contrib.postgresql.fields.ArrayField',
```

Comment:
I think as-is is fine if there's demand for it, but it does feel like giving `django.contrib.postgres` special treatment so it would be nice to have a plan for a registry-based refactor